web/src/engine/osk/src/views/anchoredOskView.ts

```diff
@@ -1,9 +1,8 @@
-import Codes from '@keymanapp/keyboard-processor/build/obj/text/codes.js';
-import DeviceSpec from '@keymanapp/web-utils/build/obj/deviceSpec.js';
+import { Codes, DeviceSpec } from '@keymanapp/keyboard-processor';
+import landscapeView from 'keyman/build/engine/dom-utils/obj/landscapeView.js';
```

Comment:
TS 4.7's `tsc` supports submodule imports, where prior versions do not.  As a result, this import path cannot be "cleaned up" further without updating to TS 4.7 _or_ giving individual engine modules their own package.json.

Node itself will happily operate with cleaner paths, such as a `'#dom-utils'` subpath defined within Web's `package.json`.  The current issue is that `tsc` at its present version will not do so.

Reply:
1. We should be going to ts 4.7 or newer for this release of Keyman.
2. I am not 100% opposed to each module having its own package.json, but it does seem to add a fair bit of noise. Benefit: black box module design -- makes module 'api' surfaces explicit,  making it much easier to refactor in the future as required, and reducing spaghetti potential.